compiler/rustc_mir_transform/src/coverage/graph.rs

```diff
@@ -9,6 +9,7 @@ use rustc_index::vec::IndexVec;
 use rustc_middle::mir::coverage::*;
 use rustc_middle::mir::{self, BasicBlock, BasicBlockData, Terminator, TerminatorKind};
 
+use rustc_middle::mir::TerminatorKind::SwitchInt;
```

Comment:
please revert this change to the previous state. Don't think that this is an improvement.

Reply:
Thanks! Has fixed it